bin/p2-verify-artifact/main.go

```diff
@@ -35,24 +36,34 @@ func main() {
 	localCopyPath := filepath.Join(dir, "temp.tar.gz")
 
 	var localCopy *os.File
-	var locationForSignature string
+	var locationForSignature url.URL
```

Comment:
Based on what I've seen in the standard library, it's idiomatic to pass around `*url.URL` values instead of `url.URL` structs directly.